sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/CommentAndTokenVisitor.java

```diff
@@ -81,8 +70,8 @@ public void visitElement(@NotNull PsiElement element) {
       Token.Type type = Token.Type.OTHER;
       if (leaf.getElementType() instanceof KtKeywordToken) {
         type = Token.Type.KEYWORD;
-      } else if (LITERAL_TOKEN_TYPES.contains(leaf.getElementType())) {
-        type = Token.Type.LITERAL;
+      } else if (leaf.getElementType() == KtTokens.REGULAR_STRING_PART) {
```

Comment:
If you decide to add character literal, you can use `KtTokens.STRINGS.contains()` here